proposals/infra/INF-0005-Long-Vector-TestPlan.md

```diff
@@ -0,0 +1,199 @@
+# Test Plan
+There are three test categories we are concerned with:
+
+1. DXIL Intrinsic tests: Go through the operators that support vectors and make a list of the DXIL ops used.
+    - The table to reference for this is in HLOperationLower.cpp.
+    - Many, if not all, of these probably have existing test cases that just need to be modified to use vectors
+        - I think that the 'implementation' will largely be editing ShaderOpArithTable.xml, which is used to generate TAEF 'data driven' tests. Minor shader code updates and added input/output of vectors
+    - Some tests may need to be duplicated for the vector variant of the test.
+    - The tests cases in this category are any intrinsics in the below table that have a DXIL::OpCode::* in the DXIL OPCode colum
+
+2. LLVM Native operation tests
+    - TBD: Where do we test these? Tex mentioned some existing tests but it sounds like we don't want to use those. And they aren't HLK tests anyways.
+    - TODO: Discuss how we want to test ops that don't boil down to a single DXIL op. Still seems like something we should test in the HLK?
```

Comment:
> Still seems like something we should test in the HLK?

Agreed.

Reply:
From some further discussion with @pow2clk on Friday we also want to test some LLVM ops. I'll add a point to this spec to capture that. But I think it would be good to also include these in the HLK.